app/javascript/modules/listingDetails/ListingDetailsAside.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react"
```

Comment:
I find it a little confusing that we have a `listingDetailsAside` module, but the `ListingDetailsAside` component is under a different module. should we move it into `listingDetailsAside`?

Reply:
Sure i can move it over.